context/condarc

```diff
@@ -0,0 +1,8 @@
+auto_update_conda: False
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - dask/label/dev
+  - pytorch
+  - conda-forge
+  - nvidia
```

Comment:
I think since these are end-user images meant for interactivity, we should use default conda behaviors including `number_channel_notices` and `always_yes`.

I know when I'm doing something in the `rapidai/ci` images, the `always_yes: true` trips me up occasionally.

Reply:
Got it, that makes sense. Let's leave it as-is and not adopt any other non-default settings.